### ISSUE TITLE
[4.0] [Front] Problem with the "background" of images

### ISSUE THREAD

**Issue by Kostelano** [joomla/joomla-cms#34546]
### Steps to reproduce the issue
In the media manager on the site and in the control panel, the "background" is displayed differently for the same images. On the site, the background of the images and the panel is merged. Just compare the two screenshots below.

I did not deeply understand the problem - perhaps it is simple. I just report that they quickly "caught" my eyes.

![Screenshot_1](https://user-images.githubusercontent.com/8440661/122441149-f01e4080-cfa5-11eb-9306-a3935d9fff7e.png)

![Screenshot_2](https://user-images.githubusercontent.com/8440661/122441158-f44a5e00-cfa5-11eb-8938-9c2da71eeacc.png)

**Comment by Kostelano** [comment]
It also looks strange when editing an image.

![Screenshot_3](https://user-images.githubusercontent.com/8440661/122441341-22c83900-cfa6-11eb-9916-8ab671660d41.png)

**Comment by brianteeman** [comment]
As with #34545 there is something missing in the css

**Comment by brianteeman** [comment]
amongst other bugs it is expecting a value being set for` :root(--hue)` example 214

**Comment by rhythmjain32** [comment]
![joomla](https://user-images.githubusercontent.com/60003625/123211276-d7031b80-d4e0-11eb-8976-418fe9520013.PNG)
Issue #34545  has been solved.

**Comment by brianteeman** [comment]
@rhythmjain32 That is a screenshot from joomla 3

**Comment by laoneo** [comment]
#34624 should fix this issue but https://github.com/joomla/joomla-cms/pull/34621 needs to be tested and merged first.

**Comment by laoneo** [comment]
Can you guys give https://github.com/joomla/joomla-cms/pull/34624 a test?

**Comment by fab966** [comment]
I use my Template (front) as default on Joomla 4.0.4, and I had assigned Cassiopeia only to a "Create Article" menu in Frontend.
**If Cassiopeia is not** the default Template the problem is still there: Media Manager has not the proper styling.
Everythings works fine if I switch back Cassiopeia as the default Template.
NOTE: Tested on some J4.0.4 and with different Templates (front).
I hope it can be helpfull.
